### PR TITLE
Release 4.0.0

### PR DESCRIPTION
**Pull request by moonso**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
 This project adheres to [Semantic Versioning](http://semver.org/).
 
-## [Unreleased]
+## [4.0.0]
 
 ### WARNING ###
 
-```
-This version will require that the backend of pre releases are updated
+This is a Major version update and will require that the backend of pre releases is updated.
```
Comment (northwestwitch) [inline CHANGELOG.md]: One understands that is Major also without capital M ! :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -20,14 +20,20 @@ $scout update hpo
 ### Fixed
 
 - Adapts clinvar parsing to new format
+- Fixed problem in `scout update user` when there where no roles
```
Comment (northwestwitch) [inline CHANGELOG.md]: Elaborate a bit (one adjective is enough) on roles.

File: CHANGELOG.md [hunk 3]
```diff
@@ -20,14 +20,20 @@ $scout update hpo
...
+- Makes pileup.js use online resources so it can be used by any instance
```
Comment (northwestwitch) [inline CHANGELOG.md]: Same here, elaborate on instance

File: CHANGELOG.md [hunk 4]
```diff
@@ -20,14 +20,20 @@ $scout update hpo
...
 
 ### Added
 
 - Filter for spidex scores
 - All HPO terms are now added and fetched from the correct source (https://github.com/obophenotype/human-phenotype-ontology/blob/master/hp.obo)
 - New command `scout update hpo`
 - New command `scout update genes` will fetch all the latest information about genes and update them
-- Load **all** variants found on chromosome **MT** 
+- Load **all** variants found on chromosome **MT**
+- Adds choice in cases owerview do display as many cases as user like
```
Comment (northwestwitch) [inline CHANGELOG.md]: is it "to display"?

File: docs/blog/new-4.0.0.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+This major update includes the following highlights:
+
+## New features
+
+- Finally!! There will always be a downloadable case report available at case page (great stuff @northwestwitch)🦄
+
+  ![Case report](/img/blog/case_report.png)
+
+- Scout now uploads ALL variants on the mitochondrial chromosome (MT) - independently of rank score both for clinical and research variants
```
Comment (northwestwitch) [inline docs/blog/new-4.0.0.md]: All variants FROM the mitochondrial chromosome (MT)

File: docs/blog/new-4.0.0.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+- Filter on spidex levels are now available
+
+- Filter on chromosome is possible, mainly to get easy access to MT variants
+
+- It is possible to produce a clinvar report from the case page that could be used to submit variants to clinvar
```
Comment (northwestwitch) [inline docs/blog/new-4.0.0.md]: It is possible to produce clinvar submission files

**Comment by moonso** [comment]
Thx, I fixed those things

**Comment by northwestwitch** [comment]
As soon as you merge I'll update stage

**Pull request merged**